server/routes/summary_routes.ts

```diff
@@ -128,15 +114,25 @@ export function registerSummaryAssistantRoutes(
         return res.ok({ body: response.body.inference_results[0].output[0].result });
       } catch (e) {
         return res.internalError();
```

Comment:
not related to this change, but we don't return 500 status code

Reply:
Which status code should we return?